packages/tracing/src/browser/metrics.ts

```diff
@@ -226,14 +234,26 @@ function addResourceSpans(
     return undefined;
   }
 
+  const data: Record<string, any> = {};
+  if ('transferSize' in entry) {
+    data.transferSize = entry.transferSize;
```

Comment:
Given that arbitrary data fields are not special-cased in the UI to make them read nice, would it be desirable here to use a "pretty" format like `data["Transfer Size"]`? cc @dashed 

Specifically:

![image](https://user-images.githubusercontent.com/88819/90629790-3ab51c00-e220-11ea-9d13-c7e47549f390.png)

Reply:
Yep that should work fine.